MaxMinInArray.java

```diff
@@ -0,0 +1,40 @@
+import java.util.ArrayList;
+import java.util.List;
+
+//TC=O(n)
```

Comment:
Since we are comparing two numbers together(i and i+1), for loop will iterate only for half the time.(i is incremented by i+2). So time complexity will be O(n/2).